.github/workflows/testing.yml

```diff
@@ -3,66 +3,66 @@ name: Testing
 on:
   pull_request:
     branches:
-    - main
+      - main
```

Comment:
Ironically, this is the syntax that is correct (in my hard opinion). But you are absolutely right that a few were wrong .. you just fixed them all the wrong way :D

Would you mind terribly to fix them back, and the ones that were broken?

As in, a list in yaml should not be intended. That makes things a lot easier to read. So the correct syntax is:

```
branches:
- main
```